server/plugin.go

```diff
@@ -265,12 +260,6 @@ func (p *Plugin) sendSubscribeConfirmationMessage(message string, subscriptionUR
 		Message:   "",
 		ChannelId: p.ChannelID,
 		UserId:    p.BotUserID,
```

Comment:
@vesari I suppose 3. is caused by the missing attachments to the post
```suggestion
		UserId:    p.BotUserID,
		Props: model.StringInterface{
			"attachments":       attachments,
		},
```

Reply:
@hanzei thanks! Committed.